linux_os/guide/system/software/updating/ensure_redhat_gpgkey_installed/oval/shared.xml

```diff
@@ -24,16 +26,24 @@
           test_ref="test_package_gpgkey-{{{ aux_pkg_version }}}-{{{ aux_pkg_release }}}_installed" />
         </criteria>
       </criteria>
-      <criteria comment="CentOS Vendor Keys" operator="OR">
+      {{%- if product == "rhel6" %}}
+      <criteria comment="CentOS Vendor Keys" operator="AND">
```

Comment:
Can you confirm this change from `OR` to `AND`. I suspect that before it was incorrect because only one of `installed_OS_is_centos6` or `test_package_gpgkey-c105b9de-4e0fd3a3_installed` was enough to make the check pass, but what we really want is that both conditions are met.

Reply:
This change to `AND` is correct.